tutor/specs/e2e/course-settings.e2e.ts

```diff
@@ -1,29 +1,28 @@
-import { Factory, Mocker, visitPage, setTimeouts } from './helpers'
+import { visitPage, setTimeouts, loginAs } from './helpers'
 
 describe('Course Settings', () => {
-    let numStudents = 5
     beforeEach(async () => {
+	await loginAs({ page, userName: 'reviewteacher' })
         await setTimeouts()
     })
-    Mocker.mock({
-        page,
-        routes: {
-            '/api/courses/:id/roster': async ({ mock, params: { id } }) => (
-                Factory.create('CourseRoster', { id, numStudents, course: mock.course(id) })
-            ),
-
-            '/api/courses/:id': async ({ request }) => {
-                if (request.method() == 'PUT') { return request.postDataJSON() as any }
-                return null
-            },
-        },
-    })
 
     describe('without any students', () => {
-
         it('deletes, then redirects to my-courses', async () => {
-            numStudents = 0
-            await visitPage(page, '/course/1/settings?tab=1')
+	    await visitPage(page, '/courses')
+	    await page.click('testEl=create-course')
+	    await page.click('[role=button].choice')
+	    await page.click('testEl=next-btn')
+	    await page.click('testEl=next-btn')
+	    await page.click('testEl=next-btn')
+
+	    await page.fill('#number-students', '1')
+	    await page.click('testEl=next-btn')
+
+	    await page.waitForSelector('.course-page')
+	    await page.click('#actions-menu')
+	    await page.click('[data-item=courseSettings]')
+
+	    await page.click('testEl=c-o-u-r-s-e-d-e-t-a-i-l-s-tab')
```

Comment:
ug, I thought that'd been fixed so it didn't dasherize the test-id :(

Reply:
Fixed, was being caused by the tab name being all uppercase